BridgeApp/BridgeApp/Data Tracking/Model/SBATrackedLoggingDataSource.swift

```diff
@@ -244,6 +244,9 @@ open class SBATrackedLoggingTableItem : RSDTableItem, RSDScheduleTime {
     /// The date when the event was logged.
     open var loggedDate: Date?
     
+    /// The timezone
```

Comment:
```suggestion
    /// The timezone in effect when the event was logged.
```
(again, if that's the intent--but either way it needs a period at the end)